model-mesh/crd/bases/serving.kserve.io_predictors.yaml

```diff
@@ -0,0 +1,233 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
```

Comment:
I think the null creationTimestamp might be necessary since listed in the additionalPrintedColumns.  I can experiment.  Annotation should be removeable